<launch>

	<node name="controller" pkg="pid" type="controller" ns="motorFR">
            <rosparam file="$(find projeto_semear)/param/pid.yaml" command="load" />
			<remap from="setpoint" to="desired_vel" />
            <remap from="control_effort" to="pwm" />
            <remap from="state" to="encoder" />
            <remap from="pid_enable" to="/pid_enable" />
    </node>

	<node name="controller" pkg="pid" type="controller" ns="motorFL">
            <rosparam file="$(find projeto_semear)/param/pid.yaml" command="load" />
			<remap from="setpoint" to="desired_vel" />
            <remap from="control_effort" to="pwm" />
            <remap from="state" to="encoder" />
            <remap from="pid_enable" to="/pid_enable" />
    </node>
    
    <node name="controller" pkg="pid" type="controller" ns="motorBR">
            <rosparam file="$(find projeto_semear)/param/pid.yaml" command="load" />
			<remap from="setpoint" to="desired_vel" />
            <remap from="control_effort" to="pwm" />
            <remap from="state" to="encoder" />
            <remap from="pid_enable" to="/pid_enable" />
    </node>
    
    <node name="controller" pkg="pid" type="controller" ns="motorBL">
            <rosparam file="$(find projeto_semear)/param/pid.yaml" command="load" />
			<remap from="setpoint" to="desired_vel" />
            <remap from="control_effort" to="pwm" />
            <remap from="state" to="encoder" />
            <remap from="pid_enable" to="/pid_enable" />
    </node>

</launch>   
